6.txt

```diff
@@ -0,0 +1 @@
+its my sixth file.

```

Comment:
OK let's pretend I am reviewing this PR ;)
Can you please make sure the line starts with a capital "I" i.e. "It's my ..."?
If you make the correction I will then merge the PR.

Reply:
I did it. Can you please merge the PR